eng/tools/TypeSpecValidation/src/rules/folder-structure.ts

```diff
@@ -18,6 +20,37 @@ export class FolderStructureRule implements Rule {
       }
```

Comment:
I believe this check should also ensure `tspconfig.yml` exists.  I believe this check would pass if the folder contained no files matching `**tspconfig.*`.

Reply:
if there's no tspconfig files, we do a noop for tsv as a whole right?

Reply:
If there is no `tspconfig.yaml`, it should be an error, not a noop.  If you call `tsv` on a folder, it's supposed to contain a `tspconfig.yaml`.  The higher-level scripts like `Validate-TypeSpec.ps1` should no-op if there is no `tspconfig.yaml`.

Reply:
I think line 67 is what you're referring to?

Reply:
Line 67 looks correct, yes.  Did you have a question?

Reply:
was confused by the original change request, the check would fail if folders did not contain tspconfig.yaml

Reply:
> was confused by the original change request, the check would fail if folders did not contain tspconfig.yaml

My original comment was based on the first commit to this PR, which I believe did not fail if the folder did not contain `tspconfig.yaml`.  This feature was added in the second commit, after my original comment.  I agree the latest code handles this correctly.